src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -289,11 +283,40 @@ public bool IsSuite
         /// <value></value>
         public int Seed { get; set; }
 
-        #endregion
+        /// <summary>
+        /// The SetUp methods.
```

Comment:
(Since neither is a sentence, I wouldn't expect to see a period at the end of either, but in any case I'd expect to see the capitalization, use of spaces, and punctuation to be consistent.)